content/creator/sdk7/3d-essentials/entity-positioning.md

```diff
@@ -395,15 +395,43 @@ When creating an `AvatarAttach` component, pass an object with the following dat
 - `avatarId`: _Optional_ The ID of the player to attach to. This is the same as the player's Ethereum address, for those players connected with an Ethereum wallet. If not speccified, it attaches the entity to the local player's avatar.
 - `anchorPointId`: What anchor point on the avatar skeleton to attach the entity, using a value from the enum `AvatarAnchorPointType`.
 
-The following anchor points are available on the player:
+The following anchor points are available on the `AvatarAnchorPointType` enum:
 
-- `AvatarAnchorPointType.AAPT_NAME_TAG`: Floats right above the player's name tag, isn't affected by the player's animations.
+- `AAPT_RIGHT_HAND`: Fixed on the player's right hand
+- `AAPT_LEFT_HAND`: Fixed on the player's left hand
+- `.AAPT_NAME_TAG`: Floats right above the player's name tag, isn't affected by the player's animations.
 
   > Note: The name tag height is dynamically adjusted based on the height of the wearables a player has on. So a player wearing a tall hat will have their name tag a little bit higher than others.
 
-- `AvatarAnchorPointType.AAPT_POSITION`: The player's overall position.
-- `AvatarAnchorPointType.AAPT_RIGHT_HAND`: Is fixed on the player's right hand
-- `AvatarAnchorPointType.AAPT_LEFT_HAND`: Is fixed on the player's left hand
+- `AAPT_POSITION`: The player's overall position.
```

Comment:
Maybe we can clarify that the position is in the ground level between the feet? And maybe also point out that currently there's a know issue where it's below the floor but will be addressed soon?

Reply:
added more detail now